### ISSUE TITLE
Update README.md to add more details about installation

### ISSUE THREAD

**Issue by snowmantw** [roma/roma#132]
I've found when I installed ROMA from source code on Ubuntu 14.04, there are some dependencies must be satisfied before setup it. Although this is just a tiny issue I still want to add it to the README file. Or, maybe there should be another file like INSTALL.

**Comment by satoryu** [comment]
Thank you for your report. 
Could you give us more detail about what additional stuff did you do for installing roma? 

I'm guessing you had to [install tokyocabinet](http://roma-kvs.org/learn/tokyocabinet_lib.html).

**Comment by snowmantw** [comment]
Yes, I'm firing a PullRequest to update the README.md about how I solved it. Because I think people would expect README.md or INSTALL under a Troubeshooting section to contain such issues.

**Comment by snowmantw** [comment]
@satoryu Sorry I don't know if ROMA has a formal code-reviewing mechanism (nethier on the README secion nor in http://roma-kvs.org/docs/developer.html), so I just follow other projects' custom to mention you as the review request.

**Comment by satoryu** [comment]
that's no problem 👍

**Comment by snowmantw** [comment]
Thanks!
